<launch>

  <param name="use_sim_time" value="false" />
  
  <!-- Start the navigation test -->
  <node pkg="srproject" type="ROSProject.py" name="Top_Project" output="screen">
    <param name="rest_time" value="10" />
    <param name="fake_test" value="false" />
  </node>


</launch>
